siliconcompiler/remote/client.py

```diff
@@ -750,10 +750,12 @@ def success_action(resp):
 
     # Print server-side version info.
     version_info = response_info['versions']
-    chip.logger.info('Software version info:')
-    chip.logger.info(f'  Server version             : {version_info["sc_server"]}')
-    chip.logger.info(f'  Server\'s SC version        : {version_info["sc"]}')
-    chip.logger.info(f'  Server\'s SC Schema version : {version_info["sc_schema"]}')
+    version_suffix = ' version'
+    max_version_string = max([len(s) for s in version_info.keys()]) + len(version_suffix)
```

Comment:
Please rename this to `max_version_string_len` I was thinking initially this was about the maximum allowed versions.
Also do we really need to have this indentation stuff? It makes the actual logic harder to read.

Reply:
it makes the version numbers line up which is what was done before and easier to scan over on the output